pip/commands/run.py

```diff
@@ -0,0 +1,20 @@
+from __future__ import absolute_import
+
+from pip.basecommand import Command, SUCCESS
+
+from pip._vendor import rwt
+
+
+class RunCommand(Command):
+    """Show help for commands"""
+    name = 'run'
+    usage = rwt.commands.help_doc
```

Comment:
To test-build the docs, I ran this command: `rwt sphinx -- -m sphinx docs out`. I didn't have to set up any virtualenv and I don't have anything to clean up after (except the `out` dir).